idaes/core/base/flowsheet_model.py

```diff
@@ -275,9 +275,10 @@ def visualize(self, model_name, **kwargs):
             **kwargs: Additional keywords for :func:`idaes.core.ui.fv.visualize()`
 
         Returns:
-            None
+            visualize_result: The :class:`idaes_ui.fv.fsvis.VisualizeResult` instance returned by :meth:`UI.visualize`
```

Comment:
After checking the docs build, I think the `visualize_result:` is being interpreted by Sphinx to be a return type rather than the name of the returned variable, which causes an extra "Return type" field to be added:

![image](https://github.com/user-attachments/assets/35ff1cce-2f4a-4e41-b19a-da19b9486371)

 To avoid this, we should remove it before merging:

```suggestion
            The :class:`idaes_ui.fv.fsvis.VisualizeResult` instance returned by :meth:`UI.visualize`
```